packages/react-components/react-table/stories/Table/Virtualized.stories.tsx

```diff
@@ -0,0 +1,236 @@
+});
+
+export const Virtualized = () => {
+  const columns: ColumnDefinition<Item>[] = React.useMemo(
+    () => [
+      createColumn<Item>({
+        columnId: 'file',
+        compare: (a, b) => {
+          return a.file.label.localeCompare(b.file.label);
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'author',
+        compare: (a, b) => {
+          return a.author.label.localeCompare(b.author.label);
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'lastUpdated',
+        compare: (a, b) => {
+          return a.lastUpdated.timestamp - b.lastUpdated.timestamp;
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'lastUpdate',
+        compare: (a, b) => {
+          return a.lastUpdate.label.localeCompare(b.lastUpdate.label);
+        },
+      }),
+    ],
+    [],
+  );
+  const {
+    getRows,
+    sort: { getSortDirection, toggleColumnSort, sort },
+  } = useTableFeatures(
+    {
+      columns,
+      items: fullItemList,
+    },
+    [useTableSort({ defaultSortState: { sortColumn: 'file', sortDirection: 'ascending' } })],
+  );
+
+  const headerSortProps = (columnId: ColumnId) => ({
+    onClick: (e: React.MouseEvent) => {
+      toggleColumnSort(e, columnId);
+    },
+    sortDirection: getSortDirection(columnId),
+  });
+
+  const rows = sort(getRows());
+  const styles = useStyles();
+
+  const rowFunc = React.useCallback(
+    (index: number) => {
+      const item: Item = rows[index].item;
+      return (
+        <TableRow key={item.file.label}>
+          <TableCell>
+            <TableCellLayout media={item.file.icon}>{item.file.label}</TableCellLayout>
+          </TableCell>
+          <TableCell>
+            <TableCellLayout
+              media={<Avatar name={item.author.label} badge={{ status: item.author.status as PresenceBadgeStatus }} />}
+            >
+              {item.author.label}
+            </TableCellLayout>
+          </TableCell>
+          <TableCell>{item.lastUpdated.label}</TableCell>
+          <TableCell>
+            <TableCellLayout media={item.lastUpdate.icon}>{item.lastUpdate.label}</TableCellLayout>
+          </TableCell>
+        </TableRow>
+      );
+    },
+    [rows],
+  );
+
+  return (
+    <div className={styles.container}>
+      <Table sortable>
+        <TableHeader>
+          <TableRow>
+            <TableHeaderCell {...headerSortProps('file')}>File</TableHeaderCell>
+            <TableHeaderCell {...headerSortProps('author')}>Author</TableHeaderCell>
+            <TableHeaderCell {...headerSortProps('lastUpdated')}>Last updated</TableHeaderCell>
+            <TableHeaderCell {...headerSortProps('lastUpdate')}>Last update</TableHeaderCell>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          <Virtualizer
+            numItems={rows.length}
+            beforeContainer={{ as: 'tr' }}
+            afterContainer={{ as: 'tr' }}
+            before={{ as: 'td' }}
+            after={{ as: 'td' }}
```

Comment:
Could you please check with @smhigley  to make sure that non-visual buffer elements should be a part of the table?

Reply:
We set `role: 'none'` on these elements via the default props which should enforce non-interactivity - will confirm this with Sarah.

Reply:
Synced with Sarah, the role: 'none' will be enough to ensure these elements do not affect table accessibility, but we will require some aria roles on the children/container at the demos level - adding those in now.

Reply:
Yes, that should be fine. If they are placed within a `<table>`, they will need to be `tr`/`td` elements since `<div>`s and other non-table elements would be kicked out by the browser's HTML parser.

Putting `role="none"` or `role="presentation"` on them is enough, or `aria-hidden="true"` if there's any non-whitespace content that needs to be hidden from the accessibility tree.